lib/galaxy_test/api/test_jobs.py

```diff
@@ -7,9 +7,9 @@
 from unittest import SkipTest
 
 import pytest
+import requests
```

Comment:
Same, I think it's good to add the header here

Reply:
It's only used to make a PUT requests to Galaxy's `/api/histories/{history_id}/contents/{item_id}` .